### DISCUSSION TITLE
Blur the page behind vomnibar

### DISCUSSION THREAD

**Discussion by Bellavene** [gdh1995/vimium-c#857] [Q&A]
Sorry that I am asking about a useless feature, but would be glad to know how to do it.
How can I blur the page in the background of opened vomnibar? And would be perfect if it will be animated.
Thank you.

**Comment by gdh1995** [comment]
You may inject CSS and JavaScript code to achieve this:
```
map <v-blur_body> toggleStyle id="v-blur-on-omni" \
  disabled=false css="body.v-omni-active{\
filter:blur(4px);transition:200ms\u0020filter\u0020linear;}"
map <v-on_omni> openUrl url="javascript:\
document.body.classList.add('v-omni-active');\
_v_omni_listened||window.addEventListener('focus',\
ev=>ev.target===window&&\
document.body.classList.remove('v-omni-active'));\
var%20_v_omni_listened=_v_omni_listened||1;"
map <v-start_omni> Vomnibar.activate
run o blur_body+$D+on_omni+100wait+start_omni
```

Then this use the `o` letter to blur a page and then show Vomnibar
- If you want to use another key, change the `o` in `run o ****`
- If you want to add parameters to Vomnibar, or use other `Vomnibar.activate***`, change the line of `map <v-start_omni> ****`
